AdSecGH/Components/3_Section/EditSection.cs

```diff
@@ -99,17 +109,19 @@ protected override void SolveInstance(IGH_DataAccess DA) {
       if (Params.Input[4].SourceCount > 0) {
         reinforcements = AdSecInput.ReinforcementGroups(this, DA, 4, true);
       } else {
-        foreach (IGroup rebarGrp in in_section.Section.ReinforcementGroups) {
+        foreach (var rebarGrp in in_section.Section.ReinforcementGroups) {
```

Comment:
Section?. use null check pls

Reply:
these file are from autoformat, out of scope for this PR